packages/vaults/src/transaction/transfer/transfer_quorum.ts

```diff
@@ -8,12 +8,35 @@ import {TransactionMapperAbstract} from "../transaction_mapper";
 import {TransactionRequest} from "../transaction_request";
 import {RequestMapperAbstract} from "../request_mapper";
 
-
+/**
+ * Interface for a transaction that transfers assets.
+ * Requires amount of approvals equal to quorum.
+ * This transaction can be created or approved by an admin.
```

Comment:
is it? member role cannot transfer assets?

Reply:
no - only admins